vscode/package.json

```diff
@@ -510,7 +510,12 @@
         },
         "konveyor.kai.providerArgs": {
           "type": "object",
-          "default": {},
+          "default": {
```

Comment:
Adding back in for run_demo support. TODO - need to allow default **and** overwriting / merging of all of these arguments when changing providers.

Reply:
Yeah I get the feeling that the standard vscode config support isn't going to be enough to handle the various permutations we want to be able to cover.  There is capability to be able to setup a custom editor (a webview really) for specific files.  So maybe in the near future, the kai server and model configs will be in a "custom" config file with a custom editor...

Reply:
@sjd78 https://github.com/konveyor/editor-extensions/pull/153 - something to get us by for now.